addons/mail/static/src/js/core/translation.js

```diff
@@ -8,11 +8,12 @@ TranslationDataBase.include({
      */
     set_bundle() {
         const res = this._super(...arguments);
-        if (owl.Component.env.services && owl.Component.env.services.messaging) {
+        const { wowlEnv } = owl.Component.env;
```

Comment:
That doesn't seem correct. owl.Component.env is the global owl environment aka the legacyEnv. The wowlEnv is not accessible from the owl object, it's created during startup and given to the webclient (which makes it available to its descendants).